### PR TITLE
Only add NSS subdir path if it exists

### PR DESCRIPTION
**Pull request by The-Compiler**
On Archlinux, the NSS package ships with /usr/lib/libnss3.so.

The auto detection works fine with that, but the fix from #3439 breaks it again
since no /usr/lib/nss/libnss3.so exists:

    INFO: Loading module hook "hook-PyQt5.QtWebEngineWidgets.py"...
    Unable to find "/usr/lib/nss/*.so" when adding binary and data files.

Thus, it should only be added on distributions where it's really necessary.
With this change, test_PyQt5_QWebEngine works on Archlinux (it failed before).

### CODE REVIEW

**Comment by bjones1** [comment]
@The-Compiler, yes I'm watching -- if all tests pass, I'll merge. Thanks for the fix!

**Comment by The-Compiler** [comment]
Err, of course that's a glob pattern, so my change is broken. Let me try again :laughing:

**Comment by bjones1** [comment]
@The-Compiler, looks good. Would you first edit your commit messages to begin with the [standard prefixes](https://pyinstaller.readthedocs.io/en/stable/development/commit-messages.html#standard-prefixes) per the developer guidelines? You might also squash the two commits.

**Comment by The-Compiler** [comment]
Whoops, missed that while skimming through them - done!

**Comment by bjones1** [comment]
Fantastic, thanks!
